docs/assets.rst

```diff
@@ -0,0 +1,673 @@
+
+.. _Bower: http://bower.io
+
+.. _pipeline-js-bundles:
+
+Building pipeline JavaScript bundles
+====================================
+Pipeline JavaScript bundles combine several JavaScript files into a single
+file, with optional minimization. For example, the file
+`static/build/js/main.js`_ is the combination of 10 JavaScript files:
+
+* `kuma/static/js/libs/jquery/jquery.js`_ (JQuery 2.2.0)
+* `kuma/static/js/libs/icons.js`_
+* `kuma/static/js/components.js`_
+* `kuma/static/js/analytics.js`_
+* `kuma/static/js/main.js`_
+* `kuma/static/js/components/nav-main-search.js`_
+* `kuma/static/js/auth.js`_
+* `kuma/static/js/highlight.js`_
+* `kuma/static/js/wiki-compat-trigger.js`_
+* `kuma/static/js/lang-switcher.js`_
+
+In development, the source files (10 for ``main.js``) are served, so there are
+10 ``<script>`` elements in the HTML when when ``{{javascript('main')}}`` is
+used in a template.  In production, the output bundle is used, so a single
+``<script>`` tag appears in the HTML. The single bundle is also processed
+with UglifyJS_, which removes whitespace, replaces variable names with
+shorter names, and performs other transformations to make the file smaller.
+See django-pipeline_ as well as the :ref:`pipeline tags <pipeline-tags>`
+section for more information.
+
+.. _`static/build/js/main.js`: https://developer.mozilla.org/static/build/js/main.js
+.. _`kuma/static/js/libs/jquery/jquery.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/libs/jquery/jquery.js
+.. _`kuma/static/js/libs/icons.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/libs/icons.js
+.. _`kuma/static/js/components.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/components.js
+.. _`kuma/static/js/analytics.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/analytics.js
+.. _`kuma/static/js/main.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/main.js
+.. _`kuma/static/js/components/nav-main-search.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/components/nav-main-search.js
+.. _`kuma/static/js/auth.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/auth.js
+.. _`kuma/static/js/highlight.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/highlight.js
+.. _`kuma/static/js/wiki-compat-trigger.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/wiki-compat-trigger.js
+.. _`kuma/static/js/lang-switcher.js`: https://github.com/mozilla/kuma/blob/master/kuma/static/js/lang-switcher.js
+.. _UglifyJS: https://github.com/mishoo/UglifyJS2/tree/v2.x
+
+.. _pipeline-css-bundles:
+
+Building pipeline CSS bundles
+=============================
+Pipeline CSS bundles are conceptually similar to
+:ref:`Pipeline JS Bundles <pipeline-js-bundles>`. Some contain multiple
+source files, such as `static/build/styles/dashboards.css`_,
+which combines:
+
+* `kuma/static/styles/dashboards.scss`_
+* `kuma/static/styles/diff.scss`_
+
```

Comment:
I like that you give specific bundle examples here and above. I'd suggest again that you link to the common.py file to specify where this bundle is defined.